src/remote/entrypoints.py

```diff
@@ -214,6 +214,14 @@ def remote_set(index: int):
     help="Enable local port forwarding. Pass value as <remote port>:<local port>. \
 If local port is not passed, the local port value would be set to <remote port> value by default",
 )
+@click.option(
+    "-h",
+    "--hot-reload",
```

Comment:
I'm not sure that `--hot-reload` is a correct name for this option, since it is not a web server. Do we have any other options? I can only think about `--stream-changes` but maybe you have better ideas

Reply:
--stream-changes is a lot better. Can't think of a better flag.